CenterForOpenScience/ember-osf-web
# [ENG-4340] Refactored all the bootstrap css out of the templates

-   Ticket: [ENG-4340]
-   Feature flag: n/a

## Purpose

Refactored all the bootstrap css out of the templates

## Summary of Changes

Refactored all the bootstrap css out of the templates

## Screenshot(s)
Carousel - Desktop - Before
![Carousel-Desktop-Before](https://user-images.githubusercontent.com/113387478/229898782-b8626b52-831c-4576-bf87-e03cc25121f0.png)

Carousel - Desktop - After
![Carousel-Desktop-After](https://user-images.githubusercontent.com/113387478/229898634-7d806cf6-f409-48e9-b34c-95446682e79c.png)

Footer - Desktop - Before
![Footer - Desktop - Before](https://user-images.githubusercontent.com/113387478/229898654-18b66f4c-0b51-4959-885b-d4f5a5499992.png)

Footer - Desktop - After
![Footer - Desktop - After](https://user-images.githubusercontent.com/113387478/229898648-62ea5a87-967f-4503-b574-36d89ae58146.png)rousel-Desktop-Before.png…]()

Footer - Mobile - Before
![Footer - Mobile - Before](https://user-images.githubusercontent.com/113387478/229898656-0d00cef0-9ac6-4f6a-9d36-ce4c3731ff93.png)

Footer - Mobile - After
![Footer Mobile - After](https://user-images.githubusercontent.com/113387478/229898657-f1780c27-d6c3-4cef-abc8-da93c8d42dbf.png)

Join Banner - Desktop - Before
![Join Banner - Desktop - Before](https://user-images.githubusercontent.com/113387478/229898661-142a507a-4e3f-4cf9-b4f9-5e08b85b375e.png)

Join Banner - Desktop - After
![Join Banner - Desktop - After](https://user-images.githubusercontent.com/113387478/229898659-22f93632-4c67-458f-8f6d-7913b40377ec.png)

Join Banner - Mobile - Before
![Join Banner - Mobile - Before](https://user-images.githubusercontent.com/113387478/229898666-927077ab-dfc0-4005-bdda-27446796369d.png)

Join Banner - Mobile - After
![Join Banner - Mobile - After](https://user-images.githubusercontent.com/113387478/229898664-0758d9dd-d3c8-4acd-8ab4-b807082616d1.png)

Password - Desktop - Before
![Password - Desktop - Before](https://user-images.githubusercontent.com/113387478/229898672-af2039e6-a643-4edb-9f2e-2334d864eeb1.png)

Password - Desktop - After
![Password - Desktop - After](https://user-images.githubusercontent.com/113387478/229898671-4e15e030-3fd9-4200-8629-1bddd093a3b1.png)

Password - Mobile - Before
![Password - Mobile - Before](https://user-images.githubusercontent.com/113387478/229898677-58552158-9175-484b-b7d5-c5c4efe130d3.png)

Password - Mobile - After
![Password - Mobile - After](https://user-images.githubusercontent.com/113387478/229898673-de8f9492-e937-4c55-a47d-4c72c36dda02.png)

Sign-up - Desktop - Before
![Sign-up - Desktop - Before](https://user-images.githubusercontent.com/113387478/229898680-ba80aa6e-5a77-4a10-ac7b-9c0d06899dec.png)

Sign-up - Desktop - Before
![Sign-up - Desktop - After](https://user-images.githubusercontent.com/113387478/229898678-0a21c9c0-a032-4937-a8a4-b201eab126dd.png)

Sign-up - Mobile - Before
![Sign-up - Mobile - Before](https://user-images.githubusercontent.com/113387478/229898683-ba66544d-75b8-4598-8d39-5fc9e4ef650f.png)

Sign-up - Mobile - Before
![Sign-up - Mobile - After](https://user-images.githubusercontent.com/113387478/229898681-92e9ce1d-4b7a-46de-acfd-f56a6a013888.png)

Registration Cards - Desktop
![Registration Cards - Desktop](https://user-images.githubusercontent.com/113387478/229913478-c944a836-34d7-428b-86cc-fc8f9e5eaba0.png)

Registration Cards - Mobile
![Registration Cards - Mobile](https://user-images.githubusercontent.com/113387478/229913492-a9537b1c-5f81-4bf6-a80e-23bb27a16983.png)



## Side Effects

Mass Chaos

## QA Notes

Check the attached images


[ENG-4340]: https://openscience.atlassian.net/browse/ENG-4340?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
